packages/suite/src/views/wallet/trading/common/TradingUtils/TradingUtilsPrice.tsx

```diff
@@ -43,12 +44,19 @@ export const TradingUtilsPrice = ({
     receiveAmount,
     receiveCurrency,
 }: TradingCryptoAmountProps) => {
-    const { type } = useTradingFormContext();
+    const { type, quotesRequest } = useTradingFormContext();
+
+    const showProviderAdjustedAmountTooltip =
+        type === 'sell' &&
+        receiveAmount &&
+        quotesRequest?.cryptoStringAmount &&
+        new BigNumber(receiveAmount).toNumber() !==
+            new BigNumber(quotesRequest?.cryptoStringAmount).toNumber();
```

Comment:
Preferring rather native method

```suggestion
   !new BigNumber(receiveAmount).isEqualTo(new BigNumber(quotesRequest?.cryptoStringAmount))
```

Reply:
done